Source/views/MacawView.swift

```diff
@@ -592,7 +592,16 @@ private class LayoutHelper {
 
     public func getTransform(_ node: Node, _ layout: ContentLayout, _ size: Size) -> CGAffineTransform {
         setSize(size: size)
-        if let rect = getNodeBounds(node: node) {
+        var rect = node.bounds
+        if let canvas = node as? SVGCanvas {
+            if let view = nodesMap.getView(canvas) {
+                rect = canvas.layout(size: view.bounds.size.toMacaw()).rect()
+            } else {
+                rect = Group(contents: canvas.contents).bounds
```

Comment:
Not really effective. It's better to use utility method to get bounds for list of nodes which will be used here and in `Group.bounds`